src/app/components/shared/demo-page.component.html

```diff
@@ -26,7 +26,13 @@ <h4 *ngIf="skyuxModulesForDisplay">
     <stache-page-anchor>
       Installation
     </stache-page-anchor>
+    <p>
+      To use <span *ngFor="let moduleName of skyuxModulesForDisplay"><stache-code>{{ moduleName }}</stache-code></span>, you must first install the module's NPM package to set it up as a dependency. In the command prompt, run the following command from your SPA's root directory:
+    </p>
     <stache-code-block [code]="npmInstall"></stache-code-block>
+    <p>
+      After the module is in place, use an Angular <stache-code>app.module.ts</stache-code> file to make it available to your SPA. For SKY UX SPAs, add <span *ngFor="let moduleName of skyuxModulesForDisplay"><stache-code>{{ moduleName }}</stache-code></span> as both an import and an export in the <stache-code>app-extras.module.ts</stache-code> file that controls the rendering of the SPA. For SKY UX libraries, add <span *ngFor="let moduleName of skyuxModulesForDisplay"><stache-code>{{ moduleName }}</stache-code></span> as an import in the module for the component that consumes it.
```

Comment:
I'd like to see if we can make the guidance more generic, but then be explicit about what to do when using our SDK.  It is also only necessary to add it to the `exports` section.  Perhaps something like...

"After installation, you need to make any exports from the module available to your SPA by adding it to the `exports` section of your SPA's main module.  If you are using SKY UX SDK, add it to the exports section of your `app-extras.module.ts` file."

Reply:
Sounds good.